components/Projects/Project.tsx

```diff
@@ -12,35 +12,38 @@ export const Project = ({ title, description, tags, author }: ProjectProps) => {
   const handleContribute = () => {
     return;
   };
+  const [innerWidth, setInnerWidth] = useState(0);
+  useEffect(() => {
+    setInnerWidth(window.innerWidth);
+  }, []);
   return (
     <div className="w-full p-2 md:p-0">
       <div
         className={`${
           mode
             ? 'text-white bg-box-color border border-[#363D45]'
             : 'text-dark-color border border-[#C9C9C9]'
-        } text-white shadow-md rounded-md items-center md:flex justify-center space-y-2`}
+        } text-white shadow-md rounded-md items-center flex-col md:flex md:flex-row justify-center space-y-2`}
       >
-        <div className="w-[250px] md:w-[500px] mx-auto h-auto p-4">
+        <div className="w-full p-2 flex justify-center items-center">
           <Image
             src={'/open-source.png'}
             alt={title}
-            height={'300'}
-            width={'300'}
-            className="object-contain  "
+            height={800}
+            width={innerWidth/1.2}
           />
         </div>
-        <div className="flex flex-col space-y-2 p-2">
+        <div className="h-full flex flex-col justify-between space-y-2 p-2">
           <h1 className="font-semibold text-xl">
             {title} - <span className="text-lg font-light">{author}</span>
           </h1>
-          <p className="text-[14px] ">{description}</p>
+          <p className="text-sm ">{description}</p>
```

Comment:
we should put a truncate property here so that the description overflowed can be shown as ....

Reply:
ok